src/Admin/AdherentAdmin.php

```diff
@@ -3,22 +3,17 @@
 namespace AppBundle\Admin;
 
 use AppBundle\Entity\Adherent;
-use AppBundle\Form\ActivityPositionType;
-use AppBundle\Form\GenderType;
+use AppBundle\Form\{ActivityPositionType, GenderType};
 use AppBundle\Intl\UnitedNationsBundle;
 use Misd\PhoneNumberBundle\Form\Type\PhoneNumberType;
-use Sonata\AdminBundle\Admin\AbstractAdmin;
-use Sonata\AdminBundle\Datagrid\ListMapper;
-use Sonata\AdminBundle\Datagrid\DatagridMapper;
-use Sonata\AdminBundle\Form\FormMapper;
-use Sonata\AdminBundle\Show\ShowMapper;
+use Sonata\AdminBundle\{
+    Admin\AbstractAdmin, Datagrid\ListMapper, Datagrid\DatagridMapper, Form\FormMapper, Show\ShowMapper
+};
 use Sonata\CoreBundle\Form\Type\DateRangePickerType;
-use Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQuery;
-use Sonata\DoctrineORMAdminBundle\Filter\CallbackFilter;
-use Sonata\DoctrineORMAdminBundle\Filter\DateRangeFilter;
-use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
-use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
-use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Sonata\DoctrineORMAdminBundle\{
```

Comment:
One `use` statement per page.

Reply:
We could have a debate :)

https://wiki.php.net/rfc/group_use_declarations

Reply:
Unfortunately there is no debate for this since we are following psr-2 standards:

http://www.php-fig.org/psr/psr-2/#namespace-and-use-declarations
> There MUST be one use keyword per declaration.